relay-dynamic-config/src/global.rs

```diff
@@ -1,9 +1,45 @@
+use relay_event_normalization::MeasurementsConfig;
 use serde::{Deserialize, Serialize};
 
 /// A dynamic configuration for all Relays passed down from Sentry.
 ///
 /// Values shared across all projects may also be included here, to keep
 /// [`ProjectConfig`](crate::ProjectConfig)s small.
-#[derive(Default, Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, Serialize, Deserialize, Default, PartialEq)]
 #[serde(default, rename_all = "camelCase")]
-pub struct GlobalConfig {}
+#[cfg_attr(feature = "jsonschema", derive(JsonSchema))]
+pub struct GlobalConfig {
+    /// Configuration for measurements normalization.
+    pub measurements: Option<MeasurementsConfig>,
+}
+
+#[cfg(test)]
+mod tests {
+
+    use relay_base_schema::metrics::MetricUnit;
+    use relay_event_normalization::{BuiltinMeasurementKey, MeasurementsConfig};
+
+    use super::GlobalConfig;
+
+    #[test]
+    fn test_deserialize_global_config() {
```

Comment:
Let's make the whole roundtrip and ensure serialization is also correct and no data is lost.

Reply:
but this does test the roundtrip, although the function name wasnt updated to reflect it, updating that now!